mutters-core/src/main/java/com/rabidgremlin/mutters/core/NoIntentMatch.java

```diff
@@ -0,0 +1,33 @@
+/* Licensed under Apache-2.0 */
+package com.rabidgremlin.mutters.core;
+
+import java.util.HashMap;
+
+/**
+ * An IntentMatch used to indicate no good match was found by an IntentMatcher.
+ * 
+ * @author rabidgremlin
+ *
+ */
+public class NoIntentMatch extends IntentMatch
+{
+  /**
+   * Creates a NoIntentMatch with empty matcher scores.
+   * 
+   */
+  public NoIntentMatch()
+  {
+    super(Intent.NONE, new HashMap<Slot, SlotMatch>(), null, new MatcherScores());
```

Comment:
would be nice if ```IntentMatch``` constructor took ```Map``` instead of ```HashMap```

Reply:
Implemented.